proof/archive.go

```diff
@@ -310,8 +332,9 @@ func genProofFileStoragePath(rootPath string, loc Locator) (string, error) {
 	assetID := hex.EncodeToString(loc.AssetID[:])
 
 	truncatedHash := loc.OutPoint.Hash.String()[:outpointTruncateLength]
-	fileName := fmt.Sprintf("%x-%s-%d%s", loc.ScriptKey.SerializeCompressed(),
-		truncatedHash, loc.OutPoint.Index, TaprootAssetsFileSuffix)
+	fileName := fmt.Sprintf("%x-%s-%d.%s",
```

Comment:
Can refactor slightly to re-use the other function used for reading the file in the first place. This way, the template string is only in one place.

Reply:
I'm not sure what you mean by "other function". The `genProofFileStoragePath` function is the only one that has the template string... This commit just adds the dot to the template string to make it a bit easier to parse for human eyes (IMO).

Reply:
I think I meant this one lol, got mixed up on diffs -- resolving